File: URI_Python_UFRR/1036.py
linha1 = input().split(" ")
a, b, c = linha1
a = float(a)
b = float(b)
c = float(c)
delta = b**2 - 4 * a * c
if delta < 0 or a == 0:
	print('Impossivel calcular')
elif delta > 0 and (a != 0):
	print('R1 = %.5f' %( (-b + (delta)**(1/2))/ (2*a) ))
	print('R2 = %.5f' %( (-b - (delta)**(1/2))/ (2*a) ))

File: URI_CPP_UFRR/1036.cpp
#include <iostream>
#include <math.h>
#include <iomanip>
using namespace std;
int main(){
    double a,b,c,delta;
    cin >> a >> b >> c;
    delta = (b*b) - 4 * a * c;
    if(delta > 0 && a != 0){
        cout << fixed << setprecision(5);
        cout << "R1 = "<< (-b + sqrt(delta))/(2*a) << endl;
        cout << "R2 = "<< (-b - sqrt(delta))/(2*a) << endl;
    }else if(delta < 0 or a == 0){
        cout << "Impossivel calcular" << endl;
    }

    return 0;
}

